spec/lib/magazine_luiza_rewards_v2/api/products_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'active_support/core_ext/object/blank'
```

Comment:
where is it being used?

Reply:
I'd not include only blank. I'd require the active_support totally in the spec_helper to enable this to all specs.